custom-elements/parser/parser-uses-registry-of-owner-document.html

```diff
@@ -121,6 +122,34 @@
     });
 }, 'HTML parser must use the registry of window.document in a document created by XMLHttpRequest');
 
+test_with_window(function (contentWindow, contentDocument) {
+    const element = define_custom_element_in_window(contentWindow, 'my-custom-element', []);
+    // document-open-steps spec doesn't match most browsers; see https://github.com/whatwg/html/issues/1698.
+    // However, as explained in https://github.com/whatwg/html/issues/1698#issuecomment-298748641
+    // the custom element registry will be replaced after document-open-steps.
+    contentDocument.write('<my-custom-element></my-custom-element>');
+
+    var instance = contentDocument.querySelector('my-custom-element');
+
+    assert_true(instance instanceof HTMLElement);
```

Comment:
This should be contentWindow.HTMLElement. (I believe it will still pass in Gecko due to Gecko's nonstandard instanceof.)